code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -1336,3 +1336,836 @@
+				target.remove_language(LANGUAGE_VAURCA)
+				S.banned = TRUE
+				host.last_action = world.time + 1 MINUTES
+			else
+				to_chat(src, SPAN_WARNING("You feel a sudden pain in your neural socket as you are unable to breach [target]'s defenses!"))
+				to_chat(target, SPAN_WARNING("[src] attempted to ban you from the Hivenet, but your Hive's defenses repelled them!"))
+				src.adjustHalLoss(20)
+				src.flash_pain(20)
+				host.last_action = world.time + 10 MINUTES
+				return
+		else if(S.shielded == SOCKET_SHIELDED)
+			if(prob(50))
+				to_chat(src, SPAN_WARNING("You feel a sudden pain in your neural socket as you are repelled by [target]'s countermeasures!"))
+				to_chat(target, SPAN_WARNING("[src] attempted to ban you from the Hivenet, but your countermeasures repelled them!"))
+				src.adjustHalLoss(15)
+				src.flash_pain(15)
+				host.last_action = world.time + 10 MINUTES
+				return
+			else
+				to_chat(src, SPAN_NOTICE("You extend your will, severing [target]'s connection to the Hivenet. Perhaps now it will learn its lesson."))
+				to_chat(target, SPAN_WARNING("You feel the thoughts of your fellow Vaurcae abruptly vanish, as [src]'s will smothers them all. In an instant, your neural socket is banned, cut off from hearing or speaking to the wider Hivenet."))
+				target.remove_language(LANGUAGE_VAURCA)
+				S.banned = TRUE
+				host.last_action = world.time + 1 MINUTES
+		else if(S.shielded == SOCKET_FULLSHIELDED)
+			to_chat(src, SPAN_WARNING("You feel a sudden pain in your neural socket as you are repelled by [target]'s countermeasures!"))
+			to_chat(target, SPAN_WARNING("[src] attempted to ban you from the Hivenet, but your countermeasures repelled them!"))
+			src.adjustHalLoss(15)
+			src.flash_pain(15)
+			host.last_action = world.time + 10 MINUTES
+			return
+
+/mob/living/carbon/human/proc/hivevoid() //Destroys a Vaurca's neural socket.
+	set name = "Void Hivenet User"
+	set desc = "Permanently sever a Vaurca of your own Hive from the Hivenet, destroying their neural socket. Mouv Ta may attempt to use this on Vaurca of another Hive, though this may have severe diplomatic consequences!"
+	set category = "Hivenet"
+	var/list/available_vaurca
+	var/list/fullname = splittext(src.name, " ")
+	var/surname = fullname[2]
+	var/obj/item/organ/internal/vaurca/neuralsocket/host = src.internal_organs_by_name[BP_NEURAL_SOCKET]
+	if(src.stat != CONSCIOUS)
+		to_chat(src, SPAN_WARNING("You are incapable of that in your current state!"))
+		return
+	if(!(all_languages[LANGUAGE_VAURCA] in src.languages) || !istype(host))
+		to_chat(src, SPAN_DANGER("Your mind is dark, unable to communicate with the Hive."))
+		return
+	if(!istype(src.internal_organs_by_name[BP_NEURAL_SOCKET], /obj/item/organ/internal/vaurca/neuralsocket/admin))
+		to_chat(src, SPAN_WARNING("You lack the authority to do this!."))
+		return
```

Comment:
"this is probably better off as a variable on the neural socket" from Matt.
unadressed?

Reply:
> "this is probably better off as a variable on the neural socket" from Matt. unadressed?

I did not notice that because github decided to hide it instead of the 30 resolved ones, hivenet usage checks have been turned into a proc